policyengine/outputs/macro/single/gov/local_areas/parliamentary_constituencies.py

```diff
@@ -48,8 +49,9 @@ def parliamentary_constituencies(
 
     result = {}
 
-    sim = simulation.selected
+    sim = simulation.selected_sim
     original_hh_weight = sim.calculate("household_weight").values
+    metric(sim)
```

Comment:
__issue:__ Potential error

If `metric` defaults to `None` above, in default cases, would this not raise an error?

Reply:
Yes! Thanks

Reply:
Removed